core/src/test/resources/log4j.properties

```diff
@@ -18,9 +18,20 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+log4j.appender.requestAppender=org.apache.log4j.varia.NullAppender
+
 log4j.logger.kafka=ERROR
 log4j.logger.org.apache.kafka=ERROR
```

Comment:
Honestly not sure. This file is mostly from 2011 and hasn't been touched since 2014. I assumed the root logger was off because people didn't want the log noise.

Avoiding the NullAppender would be nice if we're ok turning on the logging. I like to have that output in the case that I can't reproduce a failure, e.g. from Jenkins. But I figured someone set it up this way for a reason.

Re: changing those two to INFO, not really sure. I can't tell why they were set that way in the first place.